sql-plugin/src/main/spark320/scala/org/apache/spark/sql/rapids/shims/RapidsQueryErrorUtils.scala

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+
```

Comment:
Inadvertent?

Reply:
In the interest of time, I'm letting this slide as well.

Reply:
https://github.com/razajafri/spark-rapids/blob/private/SP-10437-352-support/sql-plugin/src/main/spark320/scala/org/apache/spark/sql/rapids/shims/RapidsQueryErrorUtils.scala

Reply:
Either me or github is getting confused here. Am I looking at the right file?